sentry-android-okhttp/src/test/java/io/sentry/android/okhttp/SentryOkHttpInterceptorTest.kt

```diff
@@ -38,66 +40,76 @@ class SentryOkHttpInterceptorTest {
             whenever(hub.options).thenReturn(SentryOptions())
         }
 
-        fun getSut(isSpanActive: Boolean = true, httpStatusCode: Int = 201, responseBody: String = "success"): OkHttpClient {
+        fun getSut(isSpanActive: Boolean = true, httpStatusCode: Int = 201, responseBody: String = "success", socketPolicy: SocketPolicy = SocketPolicy.KEEP_OPEN): OkHttpClient {
```

Comment:
just out of curiosity, why do we need `SocketPolicy`?

Reply:
To trigger IOException